src/bio2bel/sources/intact.py

```diff
@@ -262,16 +262,20 @@ def _add_my_row(graph: BELGraph, row) -> None:
         if relation in INTACT_INCREASES_ACTIONS:
             # oxidoreductase activity
             if relation == 'oxidoreductase activity electron transfer reaction':
+                target_mod = target.with_variants(
+                    pybel.dsl.ProteinModification(
+                        name='oxidoreductase activity',
+                        namespace='GO',
+                        identifier='0016491'
+                    )
+                )
+
                 graph.add_increases(
                     source,
-                    target,
+                    target_mod,
                     citation=pubmed_id,
                     evidence=EVIDENCE,
-                    subject_modifier=pybel.dsl.BiologicalProcess(
-                        name='oxidoreductase activity',
-                        namespace='GO',
-                        identifier='0016491'
-                    ),
+                    subject_modifier=pybel.dsl.activity(),
```

Comment:
oxidoreductase is the activity that would go with the subject. What's the effect on the protein?

Reply:
I would say that the object_modifier should be the reduction. 
Could I make a PR to add 'Red' to the [pmod_namespace](https://github.com/pybel/pybel/blob/746a2914db0ac012baed015dba869b22cb1c5c1b/src/pybel/language.py#L291)?

Reply:
Just do it ad-hoc for this script now and we can think of a better way later

Reply:
Ok, closed here: 355c40c5c7b5cfe62c9bb4cb262d749596d87be1

Reply:
Closed here 355c40c5c7b5cfe62c9bb4cb262d749596d87be1